frontend/src/actions/Comment.js

```diff
@@ -0,0 +1,77 @@
+export const GET_COMMENT_REQUEST = "GET_COMMENT_REQUEST"
+export const POST_COMMENT_REQUEST = "POST_COMMENT_REQUEST"
+export const PUT_COMMENT_REQUEST = "PUT_COMMENT_REQUEST"
+export const DELETE_COMMENT_REQUEST = "DELETE_COMMENT_REQUEST"
+export const GET_COMMENT = "GET_COMMENT"
+export const POST_COMMENT = "POST_COMMENT"
+export const PUT_COMMENT = "PUT_COMMENT"
+export const DELETE_COMMENT= "DELETE_COMMENT"
+export const FAILURE= "FAILURE"
+
+export const getCommentRequest = (index/*which meeting?*/, user/*or token? for authentication */) => {
+    return {
+        type: "GET_COMMENT_REQUEST",
+        index,
+        user
+    }
+}
+
+export const postCommentRequest = (comment, token) => {
+    return {
+        type: "POST_COMMENT_REQUEST",
+        comment,
+        token
+    }
+}
+
+export const putCommentRequest = (index, comment, token) => {
+    return {
+        type: "PUT_COMMENT_REQUEST",
+        index,
+        comment,
+        token
+    }
+}
+
+export const deleteCommentRequest = (index, token) => {
+    return {
+        type: "DELETE_COMMENT_REQUEST",
+        index,
+        token
+    }
+}
+
+export const getComment = (comments) => {

```

Comment:
getComment, putComment, postComment, deletecomment do not need to be have its own function. Saga will handle it, so I will delete these ones.